Backend/src/main/java/com/pg/programmerground/controller/PlaygroundController.java

```diff
@@ -22,38 +26,55 @@
      * 무한 스크롤 적용위한 방법? 찾기
      */
     @GetMapping("")
-    public ResponseEntity<ApiResponse<List<PlaygroundCardInfoDto>>> playgroundList() {
+    public ResponseEntity<ApiResponse<List<PlaygroundCardResponse>>> playgroundList() {
         return ResponseEntity.ok().body(new ApiResponse<>(playgroundService.getPlaygroundCardList()));
     }
 
     /**
      * playground 생성
      */
     @PostMapping("")
-    public ResponseEntity<ApiResponse<Long>> makePlayground(@Valid @RequestBody MakePlaygroundInfoDto info) throws Exception {
+    public ResponseEntity<ApiResponse<Long>> createPlayground(@Valid @RequestBody PlaygroundApi info) throws Exception {
         return ResponseEntity.ok().body(new ApiResponse<>(playgroundService.createPlayground(info)));
     }
 
     /**
      * playground 참여 신청
      */
     @PostMapping("/apply/{playgroundId}")
-    public ResponseEntity<ApiResponse<Boolean>> applyPlayground(@PathVariable Long playgroundId, @RequestBody ApplyPlaygroundDto applyPlayground) throws Exception {
+    public ResponseEntity<ApiResponse<Boolean>> applyPlayground(@PathVariable Long playgroundId, @RequestBody ApplyPlaygroundApi applyPlayground) throws Exception {
         return ResponseEntity.ok().body(new ApiResponse<>(playgroundService.applyPlayground(playgroundId, applyPlayground)));
     }
+
+    /**
+     * playground 신청 수락
+     */
+    @PutMapping("/accept-apply/{playgroundApplyId}")
```

Comment:
-로 분리하는건 약간,, 비권장 디자인방법. 

리소스 집합/리소스식별자
유형으로 리소스를 식별하고 http method로 동작을 내포하는데, 만약 마땅치 않다면 operation url 을 붙이는게 나을듯
```suggestion
    @PutMapping("/apply/{playgroundApplyId}/accept")
```

Reply:
apply가 동사라서 , applicants 가 더 맞을 듯 아니면 다른 복수 명사?

ex) 
put /applicants/1/reject
put /applicants/1/accept

applicants의 일부 데이터 (reject or accept)를 변경하는거라서 put 보다는 patch가 더 맞을 듯 하지만 ㅎㅎ 지원이 안되는 경우도 있으니 put으로 고고